### PR TITLE
added background, slider, and made home page responsive

### PR DESCRIPTION
**Pull request by dmbrown111**
Over all responsiveness requested of home page in place. One glitch in menu when moving from desktop to mobile. Background and text in place. Slider in place.

### CODE REVIEW

**Comment by WilGatlin** [comment]
Heres a couple areas that need work just from visual inspection: 
Screen Shot 2020-08-11 at 3 37 00 PM
Screen Shot 2020-08-11 at 3 36 39 PM

**Comment by WilGatlin** [comment]
Also not related to this pr but could you remove the header? We only need the nav element there for now and the header has an id of main. Really we should only have one id with a specific name per page.

File: index.html [hunk 1]
```diff
@@ -19,17 +19,28 @@
...
             </ul>
           </div>
         </div>
       </nav>
     </header>
+    <div id="bGImage">
+      <div class="b_g_text">
+        <h1>
+          RAR
+          <div id="spanOne">E</div>
```
Comment (WilGatlin) [inline index.html:35]: great job figuring this out but lol you put a div and gave it an id of span hahahaha. Just use a span!! Also "spanOne" is a terrible name and doesnt describe what it is.

File: index.html [hunk 2]
```diff
@@ -19,17 +19,28 @@
...
               <li><a href="./dropzones.html">Dropzones</a></li>
               <li><a href="#">Where to Start</a></li>
-              <li><a href="#">About</a></li>
               <li><a href="./contact.html">Contact</a></li>
             </ul>
           </div>
         </div>
       </nav>
     </header>
+    <div id="bGImage">
```
Comment (WilGatlin) [inline index.html:31]: bGimage is also a bad name. Background image for what? Better name would be bannerImage

Comment (WilGatlin) [inline index.html:31]: I want you to have a container around the whole banner called "banner" and for the image to be placed within in that banner.

File: CSS/index.css [hunk 3]
```diff
@@ -106,4 +111,103 @@ input#menuButton {
...
+}
+
+@media only screen and (max-width: 550px) {
+  .b_g_text {
+    transform: translate(-50%, 50%);
+  }
+
+  .b_g_text_two {
+    color: black;
+    transform: translate(8%, -50%);
```
Comment (WilGatlin) [inline CSS/index.css:203]: Good problem solving but this isnt the way I want you to handle this. I want you to use a banner container and relatively place the words according to that banner. The idea is we try to keep things in "components" and self contained. For instance we now have two components 1) Nav 2) Banner

File: CSS/index.css [hunk 4]
```diff
@@ -106,4 +111,103 @@ input#menuButton {
...
+  margin-top: 5px;
+}
+
+@media only screen and (max-width: 550px) {
+  .b_g_text {
+    transform: translate(-50%, 50%);
+  }
+
+  .b_g_text_two {
+    color: black;
```
Comment (WilGatlin) [inline CSS/index.css:202]: color: black is ok for quick development but I want us to use real hex values or rgb() for colors on a real project like this

File: CSS/index.css [hunk 5]
```diff
@@ -106,4 +111,103 @@ input#menuButton {
...
+  min-width: 50px;
+  max-width: 10px;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  cursor: pointer;
+  position: relative;
+  z-index: 1;
+  /* content: "/9679"; */
+  /* tried it insert circle via unicode in CSS, why didn't
```
Comment (WilGatlin) [inline CSS/index.css:187]: Hmm good question. I honestly am not sure and would have to look. Check this stack overflow and see if that gets you on the trail of the answer: https://stackoverflow.com/questions/10393462/placing-unicode-character-in-css-content-value

File: CSS/index.css [hunk 6]
```diff
@@ -106,4 +111,103 @@ input#menuButton {
...
+
+#pageSliderContainer {
+  width: 100%;
+  height: 10%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+#pageSlider {
```
Comment (WilGatlin) [inline CSS/index.css:174]: All of our buttons should have an id that includes the word button in it. PageSliderButton or PageDownButton (even more specific) would be better

File: CSS/index.css [hunk 7]
```diff
@@ -106,4 +111,103 @@ input#menuButton {
     transform: translateY(-305px);
     transition: transform 500ms;
   }
+
+  #bGImage {
+    margin-top: -213px;
```
Comment (WilGatlin) [inline CSS/index.css:116]: Answer to negative margin q: Not really Im not sure why youre doing this. Shouldnt need the margin I dont think.

File: CSS/index.css [hunk 8]
```diff
@@ -106,4 +111,103 @@ input#menuButton {
...
+  display: flex;
+  flex-direction: row;
+}
+
+h2 {
+  font-size: 80px;
+}
+.b_g_text_two {
+  left: 30%;
+  transform: translate(20%, -50%);
```
Comment (WilGatlin) [inline CSS/index.css:154]: I try to reserve transform for animations, id rather you use bottom: -50% in this case

**Review by WilGatlin — CHANGES_REQUESTED** [review]
Good job. I left some notes to work on

File: CSS/index.css (hunk 6 above)
Comment (WilGatlin) [reply CSS/index.css:174]: Actually itd be better to make your container have the button name and then make pageSlider to be pageSliderIconContainer and then make the "pointer" into "pageSliderButtonIcon" or something like that

File: CSS/index.css (hunk 7 above)
Comment (dmbrown111) [reply CSS/index.css:116]: comment it out, and shrink the page from desktop to mobile. If you have another way to fix that I'm game.

File: index.html (hunk 1 above)
Comment (dmbrown111) [reply index.html:35]: initially it was just one span property controlling the 2 blacked letters, but when shrinking to phone mobile I couldn't tell them to display the same color. so I had to make them as 2 different declarations.

File: CSS/index.css [hunk 9]
```diff
@@ -106,4 +111,103 @@ input#menuButton {
...
+  z-index: -1;
+
+  /* is this negative margin ok? */
+}
+
+.b_g_text {
+  text-align: center;
+  position: absolute;
+  font-size: 40px;
+  font-family: Arial, Helvetica, sans-serif;
```
Comment (WilGatlin) [inline CSS/index.css:137]: Not using the google font?!

Comment (dmbrown111) [reply CSS/index.css:137]: I still don't understand how to do that.

File: CSS/index.css (hunk 7 above)
Comment (WilGatlin) [reply CSS/index.css:116]: Should be able to fix this issue without it I think.

**Review by WilGatlin — APPROVED** [review]

**Pull request merged**